client/src/data/constants.tsx

```diff
@@ -4,21 +4,28 @@ import {defineMessages} from 'react-intl';
 
 // URLS
 export const DOWNLOAD_ZIP_URL = 'https://justice40-data.s3.amazonaws.com/data-pipeline/data/score/downloadable/Screening+Tool+Data.zip';
-export const FEATURE_TILE_BASE_URL = 'https://d2zjid6n5ja2pt.cloudfront.net';
+
+// This is the URL for the CDN that delivers the tile data
+export const FEATURE_TILE_BASE_URL = 'https://d3jqyw10j8e7p9.cloudfront.net';
```

Comment:
Why did this change from `d2zjid6n5ja2pt` to `d3jqyw10j8e7p9` and when will it change again?

Reply:
Thanks Tom.. this is a good question. @esfoobar-usds ?

Not sure why it changed, but right now we're looking into the ramifications of this change. One of them being CORS. My guess is that we wanted to separate the client from the map data layer?